src/js/components/List/stories/Action.js

```diff
@@ -17,7 +17,8 @@ export const Action = () => (
     <Box pad="large">
       <List
         data={data.slice(0, 10)}
-        pad={{ left: 'small', right: 'none' }}
+        pad={{ left: 'large', right: 'small' }}
+        // pad={{ left: 'small' }}
```

Comment:
Why is this comment needed?

Reply:
Thanks for pointing that! I was using this story as a playground and didn't realize this test was added to the commit. I'll remove it on the next commit. :smile: